### PR TITLE
DTMF-additional-support - add option description and example for transportType

### PR DESCRIPTION
**Pull request by acharlop**

### CODE REVIEW

**Comment by jmillan** [comment]
inband DTMFs are not supported in JsSIP

**Comment by acharlop** [comment]
just made a PR with support

**Comment by acharlop** [comment]
https://github.com/versatica/JsSIP/pull/629

File: content/documentation/3.4.x/api/session.html [hunk 1]
```diff
@@ -222,6 +222,7 @@
   "duration" => "Positive decimal @Number@ indicating the duration of the tone expressed in milliseconds. Default value is @100@.",
   "interToneGap" => "Positive decimal @Number@ indicating the interval between two tones expressed in milliseconds. Default value is @500@.",
   "extraHeaders" => "Optional @Array@ of @Strings@ with extra SIP headers for each INFO request.",
+  "transportType" => "Optional @String@ INFO or RFC2833. Default value is @INFO@",
```
Comment (ibc) [inline content/documentation/3.4.x/api/session.html]:
```suggestion
  "transportType" => "Optional @String@ 'INFO' or 'RFC2833'. Default value is 'INFO'",
```

**Review by ibc — CHANGES_REQUESTED** [review]

**Review by ibc — APPROVED** [review]

**Pull request merged**